[2024-08-16T01:48:21.364+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-16T01:48:21.389+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_reddit_pipeline.reddit_extraction manual__2024-08-16T01:48:20.357077+00:00 [queued]>
[2024-08-16T01:48:21.395+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_reddit_pipeline.reddit_extraction manual__2024-08-16T01:48:20.357077+00:00 [queued]>
[2024-08-16T01:48:21.396+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-08-16T01:48:21.404+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): reddit_extraction> on 2024-08-16 01:48:20.357077+00:00
[2024-08-16T01:48:21.414+0000] {standard_task_runner.py:64} INFO - Started process 98 to run task
[2024-08-16T01:48:21.418+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'etl_reddit_pipeline', 'reddit_extraction', 'manual__2024-08-16T01:48:20.357077+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/reddit_dag.py', '--cfg-path', '/tmp/tmpw04fng55']
[2024-08-16T01:48:21.420+0000] {standard_task_runner.py:91} INFO - Job 25: Subtask reddit_extraction
[2024-08-16T01:48:21.455+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_reddit_pipeline.reddit_extraction manual__2024-08-16T01:48:20.357077+00:00 [running]> on host 15d5e5353726
[2024-08-16T01:48:21.512+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Dinesh Saraswat' AIRFLOW_CTX_DAG_ID='etl_reddit_pipeline' AIRFLOW_CTX_TASK_ID='reddit_extraction' AIRFLOW_CTX_EXECUTION_DATE='2024-08-16T01:48:20.357077+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-16T01:48:20.357077+00:00'
[2024-08-16T01:48:21.513+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-16T01:48:21.531+0000] {logging_mixin.py:188} INFO - Connected to reddit!
[2024-08-16T01:48:22.236+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-16T01:48:22.238+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/pipelines/reddit_pipeline.py", line 12, in reddit_pipeline
    post_df=transform_data(post_df)
NameError: name 'transform_data' is not defined
[2024-08-16T01:48:22.251+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=etl_reddit_pipeline, task_id=reddit_extraction, run_id=manual__2024-08-16T01:48:20.357077+00:00, execution_date=20240816T014820, start_date=20240816T014821, end_date=20240816T014822
[2024-08-16T01:48:22.263+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 25 for task reddit_extraction (name 'transform_data' is not defined; 98)
[2024-08-16T01:48:22.284+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-08-16T01:48:22.301+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-16T01:48:22.302+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
